site/app/templates/Notifications.twig

```diff
@@ -2,13 +2,13 @@
 
 <div class="content">
     <div style="float: right; margin-bottom: 20px;">
-         <a href="{{ core.buildUrl({"component": "notification", "page": "notifications", "action": "mark_all_as_seen"}) }}" class="btn btn-primary">Mark all as seen</a>
+         <a href="{{ mark_all_as_seen_url }}" class="btn btn-primary">Mark all as seen</a>
```

Comment:
Is core still accessible here? I had this question before but could not seem to figure how this twig file had access to core even if it is a global variable.

Reply:
Not sure if I understood it correctly but:

**Is core accessible from twig file?**

Yes.

https://github.com/Submitty/Submitty/blob/5d8bc25d3c0bc38907d89f2291eeca05d31c4234/site/app/libraries/Output.php#L87

**Should we use twig to access core?**

No.

While it looks harmless to do so it is generally not an encouraged practice to mix twig template with php code (that's what twig is for). It is part of a working in progress issue (#2273) to remove that.